cmd/cluster/etcd_health.go

```diff
@@ -51,6 +53,34 @@ func (capture *logCapture) Write(p []byte) (n int, err error) {
 	return len(p), nil
 }
 
+func getKubeConfigAndClientSet() (*rest.Config, *kubernetes.Clientset, error) {
```

Comment:
As I've noted in Slack, we should leverage the existing client that is supplied by `newCmdEtcdHealthCheck` rather than initialise a new one.

One of the main issues with the initialisation code as-is is that it does not respect the `KUBECONFIG` environment variable, which means that this code could be picking up a kubeconfig different to what the SRE expects.

As this is pretty important to prevent accidental client connection confusion, I'd like to add a hold until it's addressed somehow.

/hold

Reply:
One option is to take the local kubeconfig out of the picture and supply a cluster id using one of these: https://github.com/openshift/osdctl/blob/34688d56ff7a79ca70b372e39a93ca21f4a2c756/pkg/k8s/client.go#L94-L120

It's utilized a lot in the infra node resize command https://github.com/openshift/osdctl/blob/master/cmd/cluster/resize/cmd.go since the automation needs to swap back and forth between a cluster and its Hive (and sometimes as backplane-cluster-admin)

Reply:
Nice! That would do very well here.

Reply:
Updated with the suggestions